evennia/commands/default/building.py

```diff
@@ -126,7 +128,17 @@ def parse(self):
                     aliases = [alias.strip() for alias in aliases.split(";") if alias.strip()]
                 if "/" in objdef:
                     objdef, attrs = [part.strip() for part in objdef.split("/", 1)]
-                    attrs = [part.strip().lower() for part in attrs.split("/") if part.strip()]
```

Comment:
I'm not sure I follow why this regex complexity is needed ... could we not just skip the `.lower()` here? :thinking:

Reply:
What I did here is to preserve a design choice (lowercase attribute names)

```
create box
You create a new Object: box.
set box/CasedAttribute = "foo"
Created attribute box/casedattribute [category:None] = foo
```

fixing a lack of expected functionality when coping w/ dicts

```
set box/dict_attr = {'foo': 'bar'}
Created attribute box/dict_attr [category:None] = {'foo': 'bar'}
set box/dict_attr['FOO'] = 'new'
Modified attribute box/dict_attr [category:None] = {'foo': 'new'}
examine box/dict_attr
-------------------------------------------------------------------------------
Attribute box/dict_attr [category=None] [type: <class 'dict'>]:

{'foo': 'new'}
```

so it has been necessary to extract the `key` part and mangling it separately from the rest of attribute name. Of course we could simply remove the `lower()`, I simply did not want to take the responsibility of introducing a breaking(ish) change :smile_cat:

Reply:
Ah, that's true; the rest of the attribute name does assume case-insensitivity. Fair enough, I buy that. Thanks for elaborating!
 
I would suggest adding a code comment to explain this in the code; always good to help the reader where needed. 
Also still needs unit tests for this case ;)